lib/asset-css.js

```diff
@@ -0,0 +1,189 @@
+'use strict';
+
+const { validate } = require('@podium/schemas');
+const { uriIsRelative, pathnameBuilder } = require('./utils');
+
+// Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link
+// NOTE: Only includes attributes used for loading CSS
+
+const _crossorigin = Symbol('podium:asset:css:crossorigin');
+const _pathname = Symbol('podium:asset:css:pathname');
+const _disabled = Symbol('podium:asset:css:disabled');
+const _hreflang = Symbol('podium:asset:css:hreflang');
+const _prefix = Symbol('podium:asset:css:prefix');
+const _title = Symbol('podium:asset:css:title');
+const _value = Symbol('podium:asset:css:value');
+const _media = Symbol('podium:asset:css:media');
+const _rel = Symbol('podium:asset:css:rel');
+const _as = Symbol('podium:asset:css:as');
+
+const toUndefined = value => {
+    if (value === false) return undefined;
+    if (value === '') return undefined;
```

Comment:
null stays null?

Reply:
Yes. This method is to explicitly make sure some values becomes `undefined` when running `.toJSON()` so they are excluded from the JSON file if they are not defined. We do not want to turn `null` into `undefined` in such an process.